.github/ISSUE_TEMPLATE/story.md

```diff
@@ -36,8 +36,10 @@ This issue's estimation includes completing:
 
 > ℹ️  Please read this issue carefully and understand it.  Pay [special attention](https://fleetdm.com/handbook/company/development-groups#developing-from-wireframes) to UI wireframes, especially "dev notes".
 
-<!--
+
 ## Context
+- Requestor(s): Who are the requestor(s) for this story, if any? They should be notified if the story gets de-prioritized. For customer requestors, use the `customer-xyz` label instead.
```

Comment:
This might be a better place to address this: https://fleetdm.com/handbook/company/development-groups#providing-

(see also “follow-through” bullet above it)

That page is linked from the top of the issue.

Another thing to consider: if we track external requestors in issues, it’ll be easy to accidentally reveal confidential info (customer names).  You addressed this but still, the problem is people only read the first bit.

I wish GitHub offered private custom fields on public issues.


Since this would be for internal requestors only, I suppose we could leave behind a “fill in the blank” explicitly asking for GitHub username?  Or a comment with an at-mention.  I tested at-mentions in descriptions and the mentioned person is not autonotified on close in that scenario.  I haven’t tested issue comments to

Reply:
@zhumo to be clear, I get why you want a prompt of some kind in the issue and it makes sense, just giving you some additional context to see what you think.

Reply:
While it is being addressed in the handbook above, this PR is intended to help operationalize that behavior. As in, if we are not including stakeholders, we are frequently forgetting who needs to be pinged if something gets de-prioritized. So I think while the handbook says to do it, this makes it standardized behavior.

And yes, the bullet there is intended to be overwritten with peoples' names. Could make that clearer.